api/scripts/legal-documents/add-new-legal-document-version.js

```diff
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+
+import { usecases } from '../../src/legal-documents/domain/usecases/index.js';
+import { Script } from '../../src/shared/application/scripts/script.js';
+import { ScriptRunner } from '../../src/shared/application/scripts/script-runner.js';
+
+export class AddNewLegalDocumentVersion extends Script {
+  constructor() {
+    super({
+      description: 'Add a new legal document version.',
+      permanent: true,
+      options: {
+        type: {
+          type: 'string',
+          describe: 'Type of document (ex: "TOS", "PDP")',
+          demandOption: true,
+        },
+        service: {
+          type: 'string',
+          describe: 'Associated service (ex: "pix-app", "pix-orga",...)',
+          demandOption: true,
+        },
+        versionAt: {
+          type: 'string',
+          describe: 'Version date of the legal document, format "YYYY-MM-DD", (ex: "2020-02-27")',
+          demandOption: true,
+        },
+      },
+    });
+  }
+
+  async handle({ options, logger }) {
+    let { type, service, versionAt } = options;
+
+    type = type?.trim();
+    service = service?.trim();
+    versionAt = versionAt?.trim();
```

Comment:
Avec la modification précédente on est sûr d'avoir des valeurs pour chaque argument:

```suggestion
    type = type.trim();
    service = service.trim();
    versionAt = versionAt.trim();
```